test/unit/sessions/softphone-session-handler.test.ts

```diff
@@ -302,7 +302,7 @@ describe('acceptSession()', () => {
     const startMediaSpy = jest.spyOn(mockSdk.media, 'startMedia').mockResolvedValue(createdStream as any);
 
     const mockIncomingStream = new MockStream({ audio: true });
-    const volume = mockSdk._config.defaults.audioVolume = 67;
+    const volume = mockSdk._config.defaults!!.audioVolume = 67;
```

Comment:
Codacy has a fix for the issue: [Forbidden non-null assertion.](https://app.codacy.com/gh/MyPureCloud/genesys-cloud-webrtc-sdk/pullRequest?prid=9637746)
```suggestion
    const volume = mockSdk._config.defaults!?.audioVolume = 67;
```